docker-compose.replicas-5.yml

```diff
@@ -30,6 +30,14 @@ services:
       - mongo2
       - mongo3
 
+  migration:
+    environment:
+      - MONGO_URL=mongodb://hearth:${HEARTH_MONGODB_PASSWORD}@mongo1,mongo2,mongo3,mongo4,mongo5/hearth-dev?replicaSet=rs0
+    depends_on:
+      - mongo1
+      - mongo2
+      - mongo3
+
```

Comment:
Shouldn't `mongo4` & `mongo5` be also added here?

Reply:
Yes, they should, however the `depends_on` isn't used by docker swarm (anymore?) https://stackoverflow.com/a/49284315/2560802

Reply:
The service should restart itself if the script fails right? Because it will fail in case mongo hasn't started before running the script